DirectEmployers/MyJobs
# [PD 3124] categories as checkbox

## What's this PR do?
PR changes the category selection dropdown into a static checkbox. Should not toggle just be static on the page and open

## Screenshots (if appropriate)
![screen shot 2017-04-10 at 9 29 05 am](https://cloud.githubusercontent.com/assets/22029077/24863768/2d4f661e-1dd0-11e7-84ea-cea51cbe946b.png)


## Where should the reviewer start?
Start inside of the CBO Application

## How should this be manually tested?
Pull down the repository. When you go into CBO you can either create a new CBO and go to the Add Contact form and see the categories at the bottom or you can edit an existing CBO's contacts and see the categories there as well

## Any background context you want to provide?
